packages/styleguides/docs/theme-overrides/partials/header.nunj

```diff
@@ -84,8 +84,8 @@
             </a>
           </li>
           <li class="ecl-navigation-menu__item" data-role="item">
-            <a href="/docs/flavors" class="ecl-navigation-menu__link ">
-            <span>Europa family and ECL flavors</span>
+            <a href="/docs/systems" class="ecl-navigation-menu__link ">
```

Comment:
Gives 404 this one https://deploy-preview-595--europa-component-library.netlify.com/docs/systems